src/ActionModal/components/ActionModal.vue

```diff
@@ -756,6 +760,24 @@ export default {
             }
           )
         }
+        if (this.network.network_type === "polkadot") {
+          transactionData = {
+            gasEstimate: this.estimatedFee,
```

Comment:
why not send the already calculated fees? and then in the extension switch between cosmos and polkadot?

Reply:
well, that is another possibility, but it is basically the same thing no? But yeah, ok, let's do it that way. Maybe this is too hacky

Reply:
best way would get the fees directly in the extension as this data you send is only for displaying and not the actual value. But I guess it can be like that for Polkadot as it is just an estimate anyway.